learn/file_handling/binary_files.md

```diff
@@ -0,0 +1,103 @@
+[Binary files](https://en.wikipedia.org/wiki/Binary_file) are computer files that contain data in a format that can be easily read and manipulated by machines. Binary files are important because they allow programmers to work with data in a way that is both efficient and secure. This article will discuss the basics of binary files in Python, how to read and write them, and some common use cases where they are most beneficial.  
```

Comment:
Don't put a link to external source from the 1st word of the page text. Put this link from the Binary files in the 2nd sentence.